Python/optimizer_bytecodes.c

```diff
@@ -840,6 +840,17 @@ dummy_func(void) {
         value = sym_new_unknown(ctx);
     }
 
+    op(_GET_ITER, (iterable -- iter, index_or_null)) {
+        if (sym_matches_type(iterable, &PyTuple_Type) || sym_matches_type(iterable, &PyList_Type)) {
+            iter = iterable;
+            index_or_null = sym_new_type(ctx, &PyLong_Type);
```

Comment:
Hm, this is sort of weird. We don't have a symbol for "unboxed int" in the JIT, but it really doesn't feel correct to type this as `int`. Maybe leave as unknown and we can update our lattice with unboxed C types later? It's not like this information is being used yet, anyways.

```suggestion
            index_or_null = sym_new_unknown(ctx);
```